content/zh/docs/faq/_index.md

```diff
@@ -0,0 +1,408 @@
+Helm 插件仍然使用 `$HELM_HOME` 作为 `$XDG_DATA_HOME` 的别名，以便希望将`$HELM_HOME`作为过渡环境的变量来保证向后兼容性。
+
+一些新的环境变量也通过插件环境变量来兼容以下更改：
+
+- `$HELM_PATH_CACHE` 针对缓存路径
+- `$HELM_PATH_CONFIG` 针对配置路径
+- `$HELM_PATH_DATA` 针对data路径
+
+Helm插件期望支持Heml 3 就建议使用新的环境变量。
+
+### CLI 命令重新命名
+
+为了更好地从包管理器中调整不当措辞，`helm delete`被重命名为`helm uninstall`。
+ `helm delete` 依然作为 `helm uninstall` 的别名保留， 因此其他格式也能使用。
+
+Helm 2 中为了清除版本清单，必须提供`--purge`参数。这个功能现在是默认使用的。
+为保留之前的操作行为，要使用 `helm uninstall --keep-history`。
+
+另外，其他一些重命名的命令提供了以下约定：
+
+- `helm inspect` -> `helm show`
+- `helm fetch` -> `helm pull`
+
+这些命令都保留了老的动词作为别名，因此您能够使用任意一种格式。
+
+### 自动创建namespace
+
+当创建了一个命名空间中不存在的版本时，Heml 2会创建一个命名空间。
+Heml 3中沿用了其他Kubernetes 工具的形式，如果命名空间不存在，就返回错误。
+如果您明确指定 `--create-namespace` 参数，Helm 3 会创建一个命名空间。
+
+## 安装
+
+### 为什么没有针对Fedora和其他Linux发行版的Helm原生包?
+
+Heml 项目不维护针对操作系统和环境的包。Helm 社区如果觉得需要提供原生包时，可以提供原生包。
+这是Homebrew 开始列出的方案。如果您有兴趣维护一个包，我们会很高兴的。
+
+### 为什么会提供 `curl ...|bash` 脚本?
+
+我们的仓库(`scripts/get-helm-3`)中有个脚本可以作为 `curl ..|bash` 脚本执行。
+使用HTTPS传输，The transfers are all protected by HTTPS, 并且这个脚本对它获取到包做一些审核。
+然而这个脚本有任何shell脚本的所有常见危险。
+
+我们提供这个脚本因为它很好用，但是我们建议用户先仔细阅读这个脚本。不过我们真正想要的是更好的Heml打包版本。
+
+### 我如何将Helm客户端文件放置在其他位置而不是默认位置?
+
+Helm 使用XDG结构存储文件。这些环境变量可以用来覆盖默认位置:
+
+- `$XDG_CACHE_HOME`: 设置另一个存储缓存文件的位置。
+- `$XDG_CONFIG_HOME`: 设置另一个存储Helm配置的位置。
+- `$XDG_DATA_HOME`: 设置另一个存储Helm数据的位置。
+
+注意，如果有已经存在的仓库，您需要使用 `helm repo add...` 重新添加。
+
+
+## 卸载
+
+### 我想删除我本地Helm. 全部文件在什么位置？
+
+连同 `helm` 二进制文件一起，Heml将文件存储在以下位置：
+
+- $XDG_CACHE_HOME
+- $XDG_CONFIG_HOME
+- $XDG_DATA_HOME
+
+下面这个表格按照操作系统给出了对应的默认文件夹位置：
+
+| Operating System | Cache Path                  | Configuration Path               | Data Path                 |
+|------------------|-----------------------------|----------------------------------|---------------------------|
+| Linux            | `$HOME/.cache/helm `        | `$HOME/.config/helm `            | `$HOME/.local/share/helm` |
+| macOS            | `$HOME/Library/Caches/helm` | `$HOME/Library/Preferences/helm` | `$HOME/Library/helm `     |
+| Windows          | `%TEMP%\helm  `             | `%APPDATA%\helm `                | `%APPDATA%\helm`          |
+
+## 故障排除
+
+### 在 GKE (Google Container Engine) 我遇到了 "No SSH tunnels currently open"
+
+```
+Error: Error forwarding ports: error upgrading connection: No SSH tunnels currently open. Were the targets able to accept an ssh-key for user "gke-[redacted]"?
+```
+
+另一个错误消息的形式：
+
+
+```
+Unable to connect to the server: x509: certificate signed by unknown authority
+```
+
+这个问题是说您本地的Kubernetes 配置文件需要有正确的证书。
+
+当你在GKE上创建集群时，它会为您提供一个证书，包括SSL证书和证书颁发机构。
+这些需要放在Kubernetes配置文件中（默认位置: `~/.kube/config`），保证 `kubectl` 和 `helm` 可以访问他们。
+
+### 从Helm 2迁移后， `helm list` 仅显示部分（或者不显示）我的发布版本
+
+您很可能忽略了一个事实：Helm 3 现在使用集群的命名空间来确定版本范围。 这意味着所有涉及版本的命令您都必须：
+
+* 在活动的kubernetes上下文中需要依赖当前的命名空间 (如 `kubectl config view --minify` 命令所述)，
+* 使用 `--namespace`/`-n` 参数指定正确命名空间，或者
+* 对于 `helm list` 命令，指定 `--all-namespaces`/`-A` 参数
+
+这适用于 `helm ls`、 `helm uninstall` 以及其他所有涉及版本的 `helm` 命令。
```

Comment:
This line says:
>This applies to helm ls, helm uninstall, and all other helm commands referencing a release.

The "referencing a release" part means that this statement applies to all commands that are talking about a specific release. Does the translation explain this? Thanks.

Reply:
Yes.